app/src/main/java/com/devshish/internship/presentation/ui/web/WebFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.devshish.internship.presentation.ui.web
+
+import android.os.Build
+import android.os.Bundle
+import android.view.View
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import android.widget.Toast
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.devshish.internship.R
+import com.devshish.internship.databinding.FragmentWebBinding
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class WebFragment : Fragment(R.layout.fragment_web) {
```

Comment:
Also: somehow it is just blank white screen on web view on Pixel 4A (Android 12)
Google what it could be and try to fix

Reply:
Just Create task in project and skip